ibis/backends/pandas/__init__.py

```diff
@@ -24,8 +26,13 @@ def connect(self, dictionary):
         -------
         Client
         """
-        self.client = self.client_class(backend=self, dictionary=dictionary)
-        return self.client
+        # register dispatchers
+        from . import execution  # noqa F401
```

Comment:
can these imports be done at the top?

Reply:
Unfortunately, the `ibis.backends.pandas.execution` module imports `ibis.backends.pandas` to be able to register the pandas `Backend` class in some dispatchers.  This created a lot of problems in the dispatching, and I'd be happy to try to improve that code later, since it's quite difficult to maintain. But for now this is the best solution I've found.

Same applies to the code in the other comment below.